Makefile

```diff
@@ -261,6 +261,15 @@ else
 endif
 	./scripts/test.sh "$(GO)" "$(GOFLAGS)" "$(ALL_PACKAGES)" "$(TESTS)" "$(TESTFLAGS)" "$(GOBIN)"
 
+unit-test-server: start-docker go-junit-report do-cover-file ## Runs tests.
```

Comment:
I see we do this for `test-server` already, but I'm curious how `go-junit-report` works here. Since it will get run as a dependency of `unit-test-server` before we ever `$(GO) test`, it seems like it won't have anything on which to generate a report?

Reply:
Can we tweak the help comment here from `Runs tests` to something that captures the changes here?

Reply:
Sure, we can add the help comment.

This change is for run the unit tests locally without any coverage reports and things like that. I'm not 100% sure about the need of this task by itself because you can do it manually.

Anyway, this execution task is executed in like 3 minutes and the one with the coverage is taking 6.